server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertTabletPlan.java

```diff
@@ -125,6 +125,23 @@ public void setRange(List<Integer> range) {
     return ret;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    if (paths != null) {
+      List<String> ret = new ArrayList<>();
+      for (Path path : paths) {
+        ret.add(path.getFullPath());
+      }
+      return ret;
+    }
+    List<Path> ret = new ArrayList<>();
+    for (String m : measurements) {
+      ret.add(new Path(deviceId, m));
```

Comment:
Just use `ret.add(deviceId + IoTDBConstant.PATH_SEPARATOR + m)` and return ret.

Reply:
I'll change it. Thanks for your suggestion.